package.json

```diff
@@ -66,18 +66,18 @@
     "gulp-replace": "^1.0.0",
     "gulp-rollup-lightweight": "^1.0.10",
     "gulp-sass": "^4.0.2",
-    "gulp-uglify-es": "^1.0.4",
+    "gulp-uglify-es": "^2.0.0",
```

Comment:
did we want to do major upgrades? i know we said no earlier but if we already tested it, then it LGTM, just making sure

Reply:
This is what was recommended by Dependabot. There were no more minors (or patches) on the v1 branch. I tried my best to make sure the major upgrade didn't bork us. I re-generated assets and ensured that the size was still comparable.